server/blob/blob-union/src/main/java/org/apache/james/blob/union/HybridBlobStore.java

```diff
@@ -78,24 +78,31 @@ public HybridBlobStore build() {
         public static final int DEFAULT_SIZE_THRESHOLD = 32 * 1024;
         public static final boolean DEFAULT_DUPLICATE_WRITE = false;
         public static final Configuration DEFAULT = new Configuration(DEFAULT_SIZE_THRESHOLD, DEFAULT_DUPLICATE_WRITE);
-        private static final String PROPERTY_NAME = "hybrid.size.threshold";
+        private static final String SIZE_THRESHOLD_PROPERTY = "hybrid.size.threshold";
         private static final String DUPLICATE_WRITE_PROPERTY = "hybrid.duplicate.writes";
 
         public static Configuration from(org.apache.commons.configuration2.Configuration propertiesConfiguration) {
-            return new Configuration(propertiesConfiguration.getInteger(PROPERTY_NAME, DEFAULT_SIZE_THRESHOLD),
+            return new Configuration(propertiesConfiguration.getInteger(SIZE_THRESHOLD_PROPERTY, DEFAULT_SIZE_THRESHOLD),
                 propertiesConfiguration.getBoolean(DUPLICATE_WRITE_PROPERTY, DEFAULT_DUPLICATE_WRITE));
         }
 
         private final int sizeThreshold;
         private final boolean duplicateWrite;
 
         public Configuration(int sizeThreshold, boolean duplicateWrite) {
-            Preconditions.checkArgument(sizeThreshold >= 0, "'" + PROPERTY_NAME + "' needs to be positive");
+            Preconditions.checkArgument(sizeThreshold >= 0, "'%s' needs to be positive", SIZE_THRESHOLD_PROPERTY);
 
             this.sizeThreshold = sizeThreshold;
             this.duplicateWrite = duplicateWrite;
         }
 
+        public Configuration(int sizeThreshold) {
```

Comment:
Where is that default constructor used? Do we really need it?

Reply:
It using at `BlobStoreChoosingModuleTest.java`.It is for easier testing.